rem/remApp.swift

```diff
@@ -349,15 +349,15 @@ func drawStatusBarIcon(rect: CGRect) -> Bool {
         return false
     }
     
-    private func retryScreenshot(shareableContent: SCShareableContent) {
-        if screenshotRetries < 3 {
-            screenshotRetries += 1
-            screenshotQueue.asyncAfter(deadline: .now() + 2) { [weak self] in
-                self?.scheduleScreenshot(shareableContent: shareableContent)
+    private func retryScreenCapture() {
+        if screenCaptureRetries < 3 {
+            screenCaptureRetries += 1
+            Task {
+                await startScreenCapture()
             }
```

Comment:
fwiw this removes the 2 second delay it had before. is this intentional? should the retry happen immediately?

Reply:
Good catch! I added a fix and will do some testing